docs/scie.md

```diff
@@ -0,0 +1,310 @@
+# PEX scies
```

Comment:
I think this is a really impactful feature, but users are very likely to not know what "scie" refers to, and the lede is somewhat buried, both in the sidebar and the text. 

![image](https://github.com/user-attachments/assets/3eddefbe-2d82-470b-868d-c94745b0aba4)

I think people will be able to understand and discover the feature more easily a heading like:

```suggestion
# PEX with included Python interpreter
```

Potentially the first two sentences could be flipped too, e.g.:

> PEX supports building "scie" executables that contain both a CPython interpreter and your PEX'd code, in addition to traditional [PEP-441](https://peps.python.org/pep-0441/) PEX zip file. You can create a PEX scie by adding `--scie eager` to your `pex` command line.

Reply:
Other that, I like the docs a lot 👍

Reply:
Yeah, agreed - scie means nothing still. Fixed.

Reply:
Re-rendered: https://jsirois.github.io/pex/scie.html